gatewayd_plugins.yaml

```diff
@@ -39,11 +39,7 @@ actionTimeout: 30s
 
 # action redis configures a Redis connection for the async actions to be published to.
 actionRedis:
-  # enabled controls whether to enable redis as async action queue
-  enabled: false
-
-  # if enabled, the url and channel are used to connect to the Redis server.
```

Comment:
This shouldn't be enabled for now, because the consumer is not ready yet: #472.

Reply:
Hi. I've resolved the issues, and signed my commits and other checks.